app/form/field/NumericField.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * GNU General Public License Usage
+ * This file may be used under the terms of the GNU General Public License version 3.0 as published by the Free Software Foundation and appearing in the file LICENSE included in the packaging of this file.  Please review the following information to ensure the GNU General Public License version 3.0 requirements will be met: http://www.gnu.org/copyleft/gpl.html.
+ *
+ * http://www.gnu.org/licenses/lgpl.html
+ *
+ * @description: This class provide aditional format to numbers by extending Ext.form.field.Number
+ *
+ * @author: Greivin Britton
+ * @email: brittongr@gmail.com
+ * @version: 2 compatible with ExtJS 4
+ */
+
+/***********************************
+
+GOT FROM HERE
```

Comment:
Remove this additional comment and add the link to the @description comment above. Also note that the code is "based on" the class, and has been modified to work with ExtJS 6

Reply:
done